openfl/interface/aggregator.py

```diff
@@ -91,6 +92,13 @@ def start_(plan, authorized_cols, task_group):
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
 
+    # check if remote attestation is enabled
+    if parsed_plan.config["aggregator"]["settings"].get("enable_remote_attestation", False):
+        # check if the aggregator is running in a remote attestation environment
+        attestation_utils.get_remote_attestation(parsed_plan, "aggregator")
```

Comment:
As mentioned by @psfoley , we will likely need to introduce a `fx collaborator certify --import-self-signed` command for the aggregator's attested certificate to be added to each collaborators' trust store.